config/config.yml

```diff
@@ -368,3 +368,6 @@ DUMP_COLLECTION_SEEDS: [
           "chromatic_yuletide_2010", "Fandom_Stocking", "yuletidemadness2010", "PornBattleXI",
           "Remix2011", "PornBattleXII"
 ]
+
+# How long should the numbers on the homepage be cached for
+HOMEPAGE_CACHE_TIME: 1.hour
```

Comment:
The numbers aren't the only thing on the homepage that's cached for a certain amount of time -- it might be better to give this a more specific name, like `HOMEPAGE_COUNTS_CACHE_TIME`.